lib/registries/addon/overview/template.hbs

```diff
@@ -62,8 +62,33 @@
                 <layout.leftNavOld as |leftNav|>
                     <leftNav.link data-analytics-name='Overview' @route='registries.overview.index'
                         @models={{array this.registration.id}} @icon='home' @label={{t 'registries.overview.title'}} />
-                    <leftNav.link data-analytics-name='Files' @route='registries.overview.files' @icon='file-alt'
-                        @label={{t 'registries.overview.external_links.files'}} />
+                    <leftNav.link data-analytics-name='Files' @route='registries.overview.files'
+                        @models={{array this.registration.id}} @icon='file-alt'
+                        @label={{t 'registries.overview.files.title'}} />
+                    {{#if this.onFilesRoute}}
+                        <div local-class='FileProvidersList'>
+                            {{#each this.registration.files as |provider|}}
+                                <div local-class='FileProvider'>
+                                    {{provider.name}}
+                                    {{#if (eq provider.name 'osfstorage')}}
+                                        <span>
+                                            <FaIcon
+                                                @icon='globe'
+                                            >
+                                            </FaIcon>
+                                            <BsTooltip
+                                                @placement='right'
+                                                @triggerEvents='hover'
+                                            >
+                                                {{t 'registries.overview.files.storage_location'}}
+                                                {{!-- region here --}}
```

Comment:
You should be able to get this from the registration's region relationship, but you'll have to add it to the model, it looks like. There's no separate story ticket for this, but we could add one if you don't think it's appropriate for this PR.